parser/src/java/org/apache/hadoop/hive/ql/parse/HiveParser.g

```diff
@@ -214,6 +214,7 @@ TOK_ALTERTABLE_ADDCONSTRAINT;
 TOK_ALTERTABLE_UPDATECOLUMNS;
 TOK_ALTERTABLE_OWNER;
 TOK_ALTERTABLE_SETPARTSPEC;
+TOK_ALTERTABLE_ROLLBACK;
```

Comment:
do we need to set this as a non-reserverd keyword?

Reply:
@pvary `rollback` as a keyword was already present in hive. This is just the token definition. 
@marton-bod we have discussed that option, but with the Impala team we agreed on this syntax